crates/stc_ts_file_analyzer/src/analyzer/expr/mod.rs

```diff
@@ -3053,16 +3053,22 @@ impl Analyzer<'_, '_> {
                             if let Some(vars) = exports.vars.get(sym).cloned() {
                                 return Ok(vars);
                             }
+
+                            for (id, tys) in exports.private_types.iter() {
+                                if id.sym().eq(sym) {
```

Comment:
Please use `==` instead

Reply:
Is there a particular reason other than readability?